src/components/EventEdit.tsx

```diff
@@ -232,21 +245,17 @@ class EventEdit extends React.PureComponent<PropsType> {
       },
     };
 
-    const recurring = this.props.item && this.props.item.isRecurring();
+    if (this.props.item && this.props.item.isRecurring()) {
+      console.log(1);
+    }
     const differentTimezone = this.state.timezone && (this.state.timezone !== getCurrentTimezone()) && timezoneLoadFromName(this.state.timezone);
 
+
     return (
-      <React.Fragment>
+      <>
         <h2>
           {this.props.item ? 'Edit Event' : 'New Event'}
         </h2>
-        {recurring && (
-          <div>
-            <span style={{ color: 'red' }}>IMPORTANT: </span>
-            This is a recurring event, for now, only editing the whole series
-            (by editing the first instance) is supported.
-          </div>
-        )}
```

Comment:
This warning is still needed and is still valid. We are still only editing the whole event series rather than let people edit specific instances to create recurrence exceptions.

Reply:
ill move it back to the top like before